engine/src/main/java/org/terasology/engine/TerasologyConstants.java

```diff
@@ -56,6 +56,8 @@
     /**
      * Name of the core gameplay module. The core gameplay module adds a minimal environment for some basic gameplay.
      */
+    //TODO: CSG moved out of the engine repo, so this should either be removed or reference the Builder Sample Gameplay
+    //      which still lives in the engine repo.
     public static final Name CORE_GAMEPLAY_MODULE = new Name("coresamplegameplay");
```

Comment:
```suggestion
    public static final Name CORE_GAMEPLAY_MODULE = new Name("buildersamplegameplay");
```

Reply:
That's fine by me and IIRC we have some logic somewhere else that attempts to detect JoshariasSurvival as the default when the full module lineup is present anyway.